SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRum.swift

```diff
@@ -156,6 +166,11 @@ public let DEFAULT_DISK_CACHE_MAX_SIZE_BYTES: Int64 = 25 * 1024 * 1024
      */
     @objc public var sessionSamplingRatio: Double = 1.0
 
+    /**
+     Set the time interval between batch span sampling in seconds
```

Comment:
Perhaps a more apt description would be "maximum interval between 2 consecutive span exports"